examples/grpc-server-with-prometheus/server/server.go

```diff
@@ -36,6 +36,7 @@ var (
 	})
 )
 
+// NOTE: Graceful shutdown is missing. Don't use this demo to your production.
```

Comment:
`s/to your production/in your production setup/`

Reply:
The suggestion format is like replacement in vim. Isn't is? hah

Reply:
Well quite clear IMO (: isn't?